week-3/quote-generator/style.css

```diff
@@ -1 +1,57 @@
-/** Write your CSS in here **/
+/* General body styling */
+body {
```

Comment:
I really like how the css is written. Good choice between classes and ids, comments, ...

Reply:
yes the code is more understandable with comments